### ISSUE TITLE
It’s still doing aboutblank#blocked

### ISSUE THREAD

**Issue by WP0323** [swagging-post/Blooket-Cheat-GUI#22]
Idk what to do

**Comment by swagging-post** [comment]
what website are you executing it on, plus make sure that if you're doing console, do .js and not .min.js, because min is bookmarklet, unless you're doing that

**Comment by WP0323** [comment]
Yeah I’m trying to do bookmarklet on a school Chromebook I’m going to guess it’s something with my content setting on the Chromebook?

**Comment by swagging-post** [comment]
possibly, i'll remove the forced setting links later for now, cause its kinda useless
